ramls/loan.json

```diff
@@ -93,6 +93,10 @@
       "description": "Indicates whether or not this loan had its due date modified by a recall on the loaned item",
       "type": "boolean"
     },
+    "declaredLostDate" : {
+      "description": "Date and time loan was declared lost",
+      "type": "string"
+    },
```

Comment:
do we need `"format": "date-time"` here?

Reply:
Maybe we do, but all other dates are stored as strings. Do you think we should stay consistent?

Reply:
Sure.